mon-pix/app/routes/application.js

```diff
@@ -23,6 +25,7 @@ export default Route.extend(ApplicationRouteMixin, {
   },
 
   async beforeModel(transition) {
+    this.intl.setLocale([ENV.APP.LOCALE]);
```

Comment:
```suggestion
    this.intl.setLocale([ENV.APP.LOCALE, 'fr-fr']);
```
Pour être sûr qu'il récupère fr-fr s'il ne trouve rien.

Reply:
Dans le fichier d'environnement `mon-pix/config/environment.js`, on a déjà un fallback sur `fr-fr`. Est-ce nécessaire de rajouter celui-ci au final?

Reply:
Pour moi oui.
Dans environnement, si on n'a pas préciser de langue de base, on prend 'fr-fr'.
Ici, ça précise que si le texte recherché n'existe pas dans le fichier de la Locale, on cherche dans le fichier fr-fr